### ISSUE TITLE
Add Parameters property to Pass state schema

### ISSUE THREAD

**Issue by hhprogram** [airware/asl-validator#40]
Per the [Pass state docs](https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-pass-state.html) I think a line needs to be added for `Parameters` in the [pass state schema file](https://github.com/airware/asl-validator/blob/master/src/schemas/pass.json) . I think the entry should be the same as the schema found in the [task state schema](https://github.com/airware/asl-validator/blob/master/src/schemas/task.json#L89).

I'm currently using this [plugin](https://github.com/horike37/serverless-step-functions) that relies on this validator and I am getting an error when defining my AWS step function state machine with a `Pass` state that contains a `Parameters` property. 

The error I get when trying to deploy a state machine:
```
 ✕ State machine "elasticsearch-cycle" definition is invalid:
[{"keyword":"additionalProperties","dataPath":".States['PopIPFromList']","schemaPath":
"#/additionalProperties","params":{"additionalProperty":"Parameters"},"message":"should NOT have additional properties"}, 
{"keyword":"additionalProperties","dataPath":".States['PopIPFromList']","schemaPath":
"http://asl-validator.cloud/fail#/additionalProperties","params":{"additionalProperty":"Parameters"},"message":"should NOT have additional properties"}, 
{"keyword":"additionalProperties","dataPath":".States['PopIPFromList']","schemaPath":
"#/additionalProperties","params":{"additionalProperty":"Parameters"},"message":"should NOT have additional properties"}, 
{"keyword":"additionalProperties","dataPath":".States['PopIPFromList']","schemaPath":
"http://asl-validator.cloud/pass#/additionalProperties","params": 
{"additionalProperty":"Parameters"},"message":"should NOT have additional properties"},{"keyword":"additionalProperties","dataPath":".States['PopIPFromList']","schemaPath":
"http://asl-validator.cloud/succeed#/additionalProperties","params": 
{"additionalProperty":"Parameters"},"message":"should NOT have additional properties"},{"keyword":"pattern","dataPath":".States['PopIPFromList'].Type","schemaPath":
"http://asl-validator.cloud/task#/properties/Type/pattern","params": 
{"pattern":"^Task$"},"message":"should match pattern \"^Task$\""},{"keyword":"additionalProperties","dataPath":".States['PopIPFromList']","schemaPath":
"http://asl-validator.cloud/wait#/additionalProperties","params": 
{"additionalProperty":"Parameters"},"message":"should NOT have additional properties"},{"keyword":"oneOf","dataPath":".States['PopIPFromList']","schemaPath":
"#/oneOf","params":{"passingSchemas":null},"message":"should match exactly one schema in oneOf"}]
```

A snippet of my state machine:
```
          PopIPFromList:
            Type: Pass
            Parameters: 
              inputList.$: $.inputList[1:]
            End: true
```

However, this should work as this translates to the proper JSON that we see in this [example](https://docs.aws.amazon.com/step-functions/latest/dg/sample-project-transfer-data-sqs.html#sample-sqs-code-examples) (search for their `Pop Element from List` state)

Please let me know if I'm missing something or if not - how I might help push up this minor patch.

**Comment by ChristopheBougere** [comment]
Looks like a missing property 👍
Would you mind creating a PR with the fix in [the schema](https://github.com/airware/asl-validator/blob/master/src/schemas/pass.json) fix and a new `valid-pass-state.json` with a `Pass` state containing a `Parameters` key?

**Comment by hhprogram** [comment]
@ChristopheBougere - please see [PR 42](https://github.com/airware/asl-validator/pull/42) - this should address this issue.

Apologies for the opened and closed [PR 41](https://github.com/airware/asl-validator/pull/41) I didn't properly run `npm run test` locally to catch my slight typo and then I closed the PR, thinking I could re-open it once it was fixed. Hope it's not too much of an annoyance.

Anyways, let me know if there are any questions or concerns. Appreciate your prompt response.

**Comment by ChristopheBougere** [comment]
No problem for the closed PR.
Your fix is released under [1.4.2](https://www.npmjs.com/package/asl-validator/v/1.4.2) (you might have to upgrade the version in serverless-step-functions as well).
